docker-run.sh

```diff
@@ -16,10 +16,4 @@
 # limitations under the License.
```

Comment:
The file misses the shebang.

Reply:
Maybe calling the file `docker-build.sh` makes more sense, since it's calling the `build.sh` script, and this script wraps it in docker?

Reply:
@sjwiesman introduced the Makefile like 2 weeks ago. I don't care if it is a makefile or not, but I was wondering if he had a strong reason to add a Makefile.

Reply:
No strong reason, I like makefiles :shrug: 

Either solution is fine with me

Reply:
I just found it better fitting since we only do parameter passing in the end. I found that the intend of make does not include parameter passing. Is this a correct conclusion? I didn't want to include a third target to support another parameter.